design/gateway/gateway_caching.md

```diff
@@ -0,0 +1,85 @@
+# Ten Gateway Caching Design
+
+## 1. Why cache requests on Gateway?
+
+Currently, all `eth_` requests that hit the gateway are forwarded to our Sequencer. 
```

Comment:
the sequencer is temporary. Let's say to the Ten Nodes and are executed by the Ten enclave